### ISSUE TITLE
Bottom of Group Bracket is Not Rendered

### ISSUE THREAD

**Issue by MutantPlatypus** [wallento/wavedrompy#10]
The bottom curve of the bracket for a group is not rendered.

```
svg = wavedrom.render("""
{ "signal": [
  {   "name": "clk",    "wave": "p..Pp..P"},
  ["Master",
    ["ctrl",
      {"name": "write", "wave": "01.0...."},
      {"name": "read",  "wave": "0...1..0"}
    ],
    {  "name": "addr",  "wave": "x3.x4..x", "data": "A1 A2"},
    {  "name": "wdata", "wave": "x3.x....", "data": "D1"   }
  ],
  {},
  ["Slave",
    ["ctrl",
      {"name": "ack",   "wave": "x01x0.1x"}
    ],
    {  "name": "rdata", "wave": "x.....4x", "data": "Q2"}
  ]
]}
""")

svg.saveas('demo1.svg')
```

Gives

![image](https://user-images.githubusercontent.com/1389865/63871696-5ded1880-c98a-11e9-9410-c3dd19b14598.png)

Where WaveDrom Editor renders it ok:
![image](https://user-images.githubusercontent.com/1389865/63871968-e2d83200-c98a-11e9-9bcb-ce77e808b997.png)

Tested on both 1.8.0post4 and 1.8.0.post5.dev4+gc5e792a

**Comment by wallento** [comment]
Thanks for reporting! Will fix it tomorrow, I hope.

**Comment by wallento** [comment]
Fixed in f783fcd

Thanks again for reporting!
